### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by MichelDiz**
- Adding improvements to the Issues tracker system (altering some things). 
- With 3 issue templates.
- Ref: https://blog.github.com/2018-05-02-issue-template-improvements/

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/2531)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+<!-- Please search existing issues to avoid creating duplicates. -->
+<!-- You can search for more info on https://discuss.dgraph.io too. -->
+<!-- Also please test using the latest Dgraph Version. -->
```
Comment (manishrjain) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Better to ask if they have tested with the latest version.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+4. See error
+
+**Expected behavior**
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Screenshots**
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+**Additional context**
+- Have you tried reproducing the issue with latest release?
```
Comment (manishrjain) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Move this up. This is less of a question, and more of a suggestion that they should do this.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+<!-- Please search existing Feature request to avoid creating duplicates. -->
+<!-- It is recommended that you start a discussion first in: https://discuss.dgraph.io
+before open a Feature request. That way we keep the Issues track system cleaner and productive. -->
+
+<!-- Fill out below. -->
+
+**Is your feature request related to a problem? Please describe.**
+<!-- Tip: Add a Bug Report reference or a discussion in the https://discuss.dgraph.io -->
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (manishrjain) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Let's model this on Go experience reports. They're the best way to figure out what features we should be building.

https://github.com/golang/go/wiki/ExperienceReports

Can you modify the template to reflect questions there?

Comment (MichelDiz) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Sure, I'l do.

**Pull request closed without merging**